.eslintrc

```diff
@@ -0,0 +1,202 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+
+  "env": {
+    "node": true
+  },
+
+  "rules": {
+    "comma-dangle": [2,"never"],
+    "no-cond-assign": 2,
+    "no-console": 0,
```

Comment:
why was `no-console` change to `0`.  In **website-frontend** it was set to [`2`](https://github.com/ProvidenceGeeks/website-frontend/blob/master/.eslintrc#L20)

For the sake of this PR, it's ok if we add `// eslint-disable-line no-console`